src/fixate/drivers/dmm/helper.py

```diff
@@ -14,35 +13,22 @@ def open():
     return open_visa_instrument("DMM")
 
 
-class DMM(metaclass=ABCMeta):
+class DMM:
     REGEX_ID = "DMM"
     is_connected = False
 
-    def __init__(self, instrument):
-        self.instrument = instrument
-        self.samples = 1
-
-    @abstractmethod
-    def measure(self, *mode, trigger=True, **mode_params):
-        pass
-
-    @abstractmethod
     def measurement(self):
```

Comment:
Raise NotImplementedError instead of pass, now that we're not using @abstractmethod